hasura/migrations/default/1646251192439_create_table_public_vault_transactions/up.sql

```diff
@@ -0,0 +1,40 @@
+CREATE TABLE "public"."vault_transactions"
+(
+  "id" bigserial NOT NULL,
+  "name" varchar NOT NULL,
+  "date_type" varchar NOT NULL,
+  "posneg" varchar NOT NULL,
+  "trx_number" varchar NOT NULL,
+  "active_users" varchar NOT NULL,
+  "value" varchar NOT NULL,
+  "date" Date NOT NULL,
+  "vault_id" bigint NOT NULL,
+  "created_by" bigint NOT NULL,
```

Comment:
```suggestion
  "created_by" bigint,
```

we might have vault transactions created from outside the system